utils/debug.py

```diff
@@ -0,0 +1,63 @@
+# stdlib
+import inspect
+import os
+from pprint import pprint
+import sys
+
+# datadog
+from config import get_checksd_path, get_confd_path
+from util import get_os
+
+
+def run_check(name, path=None):
+    """
+    Test custom checks on Windows.
+    """
+    # Read the config file
+
+    confd_path = path or os.path.join(get_confd_path(get_os()), '%s.yaml' % name)
+
+    try:
+        f = open(confd_path)
+    except IOError:
+        raise Exception('Unable to open configuration at %s' % confd_path)
+
+    config_str = f.read()
+    f.close()
+
+    # Run the check
+    check, instances = get_check(name, config_str)
+    if not instances:
+        raise Exception('YAML configuration returned no instances.')
+    for instance in instances:
+        check.check(instance)
+        if check.has_events():
+            print "Events:\n"
+            pprint(check.get_events(), indent=4)
+        print "Metrics:\n"
+        pprint(check.get_metrics(), indent=4)
+
+
+def get_check(name, config_str):
+    from checks import AgentCheck
```

Comment:
to avoid cyclic dependencies, i.e.

1. `checks/__init__.py` imports `debug.py` to load
2. `debug.py` imports `checks/__init__.py` to load
3. `checks/__init__.py` imports `debug.py` ...